scripts/sfpshow

```diff
@@ -580,6 +580,9 @@ class SFPShow(object):
             return pm_prefix.replace('_', '')
 
     def beautify_pm_field(self, prefix, field):
+        if type(field) == bool:
```

Comment:
>bool [](http://example.com/codeflow?start=26&length=4)

Your code change is larger than "prefec_ber". You may want to update the PR title?

Reply:
Is it possible to combine your new code into existing if-else? It may improve readability.